diagnostic_rt_qpcr/libraries/diagnosticrtqpcrcompositions/source.rb

```diff
@@ -20,13 +20,19 @@ def build_stripwell_primer_probe_compositions(operations:)
     operations.each do |op|
       options = op.temporary[:options]
       program_name = options[:program_name]
-
-      stripwells = find_stripwells(sample_names: options[:sample_names],
-                                   stripwell_ot: options[:object_type],
-                                   rows: op.temporary[:microtiter_plate]
-                                                 .collection.dimensions[0])
-      if stripwells.any?(&:empty?)
-        op.error(ProtocolError, 'No enough parts in inventory')
+      given_stripwells = options[:stripwell_id]
+      stripwells = []
+      if given_stripwells
+        stripwells = given_stripwells.map { |id| Collection.find(id) }
```

Comment:
Should be able to do `Collection.find(given_stripwells)`

Reply:
Done